processes/security_release_process.md

```diff
@@ -71,6 +71,9 @@ a security release.
   * In the docker-node repository run the
     [generate-stackbrew-library.sh]( https://github.com/nodejs/docker-node/blob/master/generate-stackbrew-library.sh)
     script and replace official-images/library/node with the output generated.
+
+        $ ./generate-stackbrew-library.sh > .../official-images/library/node
```

Comment:
do we want to add that in a ``` block since this is a markdown file?

Reply:
The 8 spaces do the right thing, see https://github.com/nodejs/security-wg/blob/18b2db796951903babc4ff9d12ae0db434813f66/processes/security_release_process.md#release-day

Reply:
Oh TIL! 👌